FE/src/app/room/room.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-room',
+  templateUrl: './room.component.html',
+  styleUrls: ['./room.component.css'],
+ 
+})
+export class RoomComponent {
+ public activeIndex: number = -1;
+ public card_count: number[] = [1,2,3,4,5,6,7,8,9,10]; 
```

Comment:
[1,2,3,4,5,6,7,8,9,10] - Put this in a constants file. So that we can have multiple options in the future for the scrum points series.